### PR TITLE
Documentation : Update formatting of single animal guide

### PR DESCRIPTION
**Pull request by C-Achard**
### Motivation

The single animal guide has some older formatting present that could be improved in order to help navigation in the document, and better focus the attention of users on important concepts.

### Scope

Refreshes the guide's format to be more MyST-like, more consistent, and provide better navigation with a contents table and working links, as well as more bullet lists and consistent section headers throughout.

### TODO

- [x] Finish converting images to figures with proper refs
- [x] Fix markdown links using `[]()` to be proper  `{ref}` directives.

### CODE REVIEW

File: docs/standardDeepLabCut_UserGuide.md [hunk 1]
```diff
@@ -214,14 +320,18 @@ frames using k-means, where each frame is treated as a vector. Frames from diffe
...
 computational complexity.
 
-**CRITICAL POINT:** It is advisable to extract frames from a period of the video that contains interesting
+```{important}
+It is advisable to extract frames from a period of the video that contains interesting
 behaviors, and not extract the frames across the whole video. This can be achieved by using the start and stop
-parameters in the config.yaml file. Also, the user can change the number of frames to extract from each video using
-the numframes2extract in the config.yaml file.
+parameters in the config.yaml file.
+Also, the user can change the number of frames to extract from each video using the numframes2extract in the config.yaml file.
```
Comment (deruyter92) [inline docs/standardDeepLabCut_UserGuide.md]:
```suggestion
Also, the user can change the number of frames to extract from each video using the numframes2pick in the config.yaml file.
```

File: docs/standardDeepLabCut_UserGuide.md [hunk 2]
```diff
@@ -713,10 +902,16 @@ dynamic: triple containing (state, detectiontreshold, margin)
     given the movement of the animal).
 ```
 
+______________________________________________________________________
+
 ### (J) Filter Pose Data
 
+#### Overview
+
 You can also filter the predictions with a median filter (default) or with a [SARIMAX model](https://www.statsmodels.org/dev/generated/statsmodels.tsa.statespace.sarimax.SARIMAX.html), if you wish. This creates a new .h5 file with the ending *\_filtered* that you can use in create_labeled_data and/or plot trajectories.
```
Comment (deruyter92) [inline docs/standardDeepLabCut_UserGuide.md]:
```suggestion
You can also filter the predictions with a median filter (default) or with a [SARIMAX model](https://www.statsmodels.org/dev/generated/statsmodels.tsa.statespace.sarimax.SARIMAX.html), if you wish. This creates a new .h5 file with the ending *\_filtered* that you can use in create_labeled_video and/or plot trajectories.
```

File: docs/standardDeepLabCut_UserGuide.md [hunk 3]
```diff
@@ -841,6 +1064,8 @@ deeplabcut.create_labeled_video(
 )
 ```
 
+#### Skeleton configuration
+
 To draw a skeleton, you need to first define the pairs of connected nodes (in the `config.yaml` file) and set the
 skeleton color (in the `config.yaml` file). There is also a GUI to help you do this, use by calling
 `deeplabcut.SkeletonBuilder(configpath)`!
```
Comment (deruyter92) [inline docs/standardDeepLabCut_UserGuide.md]:
```suggestion
`deeplabcut.SkeletonBuilder(config_path)`!
```

File: docs/standardDeepLabCut_UserGuide.md [hunk 4]
```diff
@@ -713,10 +902,16 @@ dynamic: triple containing (state, detectiontreshold, margin)
     given the movement of the animal).
```
Comment (deruyter92) [inline docs/standardDeepLabCut_UserGuide.md]:
```suggestion
fraction, you can crop around your animal/object to make processing speeds faster. For example, if you have a large open
field experiment but only track the mouse, this will speed up your analysis (also helpful for real-time applications).
To use this simply add `dynamic=(True,.5,10)` when you call `analyze_videos`.

```python
dynamic: triple containing (state, detectionthreshold, margin)

    If the state is true, then dynamic cropping will be performed.
    That means that if an object is detected (i.e., any body part > detectionthreshold),
    then object boundaries are computed according to the smallest/largest x position and
    smallest/largest y position of all body parts. This window is expanded by the margin
    and from then on only the posture within this crop is analyzed (until the object is lost;
    i.e., < detectionthreshold). The current position is utilized for updating the crop window
    for the next frame (this is why the margin is important and should be set large enough
    given the movement of the animal).
```

File: docs/standardDeepLabCut_UserGuide.md [hunk 5]
```diff
@@ -248,34 +361,51 @@ class: dropdown
...
 
+```{important}
 **CRITICAL POINT:** It is advisable to **consistently label similar spots** (e.g., on a wrist that is very large, try
 to label the same location). In general, invisible or occluded points should not be labeled by the user. They can
 simply be skipped by not applying the label anywhere on the frame.
+```
+
+#### Optional addition of more labels
 
 OPTIONAL: In the event of adding more labels to the existing labeled dataset, the user need to append the new
```
Comment (deruyter92) [inline docs/standardDeepLabCut_UserGuide.md]:
```suggestion
OPTIONAL: In the event of adding more labels to the existing labeled dataset, the user needs to append the new
```

File: docs/standardDeepLabCut_UserGuide.md [hunk 6]
```diff
@@ -82,44 +110,77 @@ deeplabcut.create_new_project(
...
+└── config.yaml
+```
+
+The purpose of the subdirectories is as follows:
+
+1. **dlc-models** and **dlc-models-pytorch** have a similar structure; the first contains files for the TensorFlow engine
+   while the second contains files for the PyTorch engine. At the top level in these directories, there are directories
+   referring to different iterations of label refinement (see below): **iteration-0**, **iteration-1**, etc.
+   The iteration directories store shuffle directories, where each shuffle directory stores model data related to a
+   particular experiment: trained and tested on a particular training and testing sets, and with a particular model
```
Comment (deruyter92) [inline docs/standardDeepLabCut_UserGuide.md]:
```suggestion
   particular experiment: trained and tested on a particular training and testing set, and with a particular model
```

File: docs/standardDeepLabCut_UserGuide.md [hunk 7]
```diff
@@ -357,7 +518,7 @@ and `augmenter_type` when you call the function.
   suggest seeing our [dedicated documentation on models](dlc3-architectures) for more information (
```
Comment (deruyter92) [inline docs/standardDeepLabCut_UserGuide.md:530]:
```suggestion
and `augmenter_type` when you call the function.

- Networks: ImageNet pre-trained networks OR SuperAnimal pre-trained network weights will be downloaded, as you
  select. You can decide to do transfer-learning (recommended) or "fine-tune" both the backbone and the decoder head. We
  suggest seeing our [dedicated documentation on models](dlc3-architectures) for more information (
```

File: docs/standardDeepLabCut_UserGuide.md [hunk 8]
```diff
@@ -770,28 +970,45 @@ class: dropdown
...
 
+#### Overview
+
 The plotting components of this toolbox utilizes matplotlib. Therefore, these plots can easily be customized by
 the end user. We also provide a function to plot the trajectory of the extracted poses across the analyzed video, which
 can be called by typing:
 
-```
-deeplabcut.plot_trajectories(config_path, [‘fullpath/analysis/project/videos/reachingvideo1.avi’])
+#### Code example
```
Comment (deruyter92) [inline docs/standardDeepLabCut_UserGuide.md]:
```suggestion
#### Overview

The plotting components of this toolbox utilize matplotlib. Therefore, these plots can easily be customized by
the end user. We also provide a function to plot the trajectory of the extracted poses across the analyzed video, which
can be called by typing:

#### Code example
```

File: docs/standardDeepLabCut_UserGuide.md [hunk 9]
```diff
@@ -248,34 +361,51 @@ class: dropdown
...
+
 ### (D) Label Frames
 
+#### Overview
+
 The toolbox provides a function **label_frames** which helps the user to easily label
 all the extracted frames using an interactive graphical user interface (GUI). The user
 should have already named the bodyparts to label (points of interest) in the
 project’s configuration file by providing a list. The following command invokes the
 napari-deeplabcut labelling GUI. Checkout the \[napari-deeplabcut docs\](file:napari-gui-landing) for
```
[the remark this answers is not in the document]
Comment (deruyter92) [reply docs/standardDeepLabCut_UserGuide.md]: Seems like a valid concern? This occurs multiple times, maybe worth addressing if that is possible.

**Review by deruyter92 — APPROVED** [review]

File: docs/standardDeepLabCut_UserGuide.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (C-Achard) [reply docs/standardDeepLabCut_UserGuide.md]: Should be good now !

File: docs/standardDeepLabCut_UserGuide.md [hunk 10]
```diff
@@ -337,7 +338,7 @@ deeplabcut.extract_frames(config_path, "manual")
 ```
 
 The user can use the *Load Video* button to load one of the videos in the project configuration file, use the scroll
-bar to navigate across the video and *Grab a Frame* (or a range of frames, as of version 2.0.5) to extract the frame(s).
+bar to navigate across the video and grab a frame or a range of frames to extract the frame(s).
```
Comment (C-Achard) [inline docs/standardDeepLabCut_UserGuide.md]: Note to self: the napari plugin extraction does not support ranges. Should be added.

**Comment by C-Achard** [comment]
Thanks for the further updates @deruyter92

**Review by C-Achard — COMMENTED** [review]
Great changes, thanks! Can't approve as it is my own PR, but looks good.
